index.js

```diff
@@ -2,6 +2,7 @@
 
 const PATH_SEPARATOR = '.';
 const TARGET = '[[target]]';
+const UNSUBSCRIBE = '[[unsubscribe]]';
```

Comment:
Unless I'm missing something, it seems like it would be better to use a `Symbol` here.